### ISSUE TITLE
Add more options for ollama

### ISSUE THREAD

**Issue by mdietrich16** [dustinblackman/oatmeal#55]
I would love move options for using ollama, like the `keep_alive` to load models indefinitely or unload them immediately or the context size for the model. These additional options are specified in [the ollama github](https://github.com/ollama/ollama/blob/main/docs/api.md#parameters) and some can be set via a dedicated modelfile, but a lot of times I would want a single-shot parameter addition and not yet another modelfile, which also clutters the list of models.

**Comment by neuhaus** [comment]
Yes, also please add an option to set a Authorization header for Ollama for those who put Ollama behind a reverse proxy that adds such a check.
